app/partials/states/account_settings.html

```diff
@@ -311,6 +311,10 @@ <h2 class='subheader'>Editing another user's Isaac account.</h2>
                             </div>
                         </div>
                     </div>
+                    <div ng-if="!user.hasSegueAccount">
+                        <p>You do not currently have a password set for this account, you are signed in using <span ng-repeat="account in user.linkedAccounts"><span ng-if="$middle">, </span><span ng-if="user.linkedAccounts.length > 1 && $last"> or </span>{{account | capitalize}}</span>.</p>
+                        <p>Please <a ui-sref="pages({id: 'contact_us_teacher'})">contact us</a> if you would like to add password access to your account.</p>
```

Comment:
OK thanks, good spot. I copied another "contact us" link and must have not thought to change it, oops.